cli/script/command-executor.ts

```diff
@@ -1223,6 +1362,43 @@ function requestAccessToken(): Promise<string> {
     });
 }
 
+export var runReactNativeBundleCommand = (entryFile: string, outputFolder: string, platform: string, sourcemapOutput: string): Promise<void> => {
+    var reactNativeBundleArgs = [
+        path.join("node_modules", "react-native", "local-cli", "cli.js"), "bundle",
+        "--assets-dest", outputFolder,
+        "--bundle-output", path.join(outputFolder, "main.jsbundle"),
+        "--dev", false,
```

Comment:
is the default value for this flag `true` for `react-native` command? If not remove it.